templates/workloads/services/lb-web/manifest.yml

```diff
@@ -5,6 +5,9 @@
 # Your service name will be used in naming your resources like log groups, ECS services, etc.
 name: {{.Name}}
 type: {{.Type}}
+{{- if .Platform}}
+platform: {{.Platform}}
+{{- end}}
```

Comment:
can we move this block below `memory` in all manifests? 🙏 
I see this as a task-level configuration so we can group them together.